allcommands.ps1

```diff
@@ -92036,7 +92036,7 @@ function Get-XpDirTreeRestoreFile {
     Write-Message -Level Verbose -Message "Connecting to $SqlInstance"
     $server = Connect-SqlInstance -SqlInstance $SqlInstance -SqlCredential $SqlCredential
 
-    if (($path -like '*.bak') -or ($path -like '*trn')) {
+    if ((($path -like '*.bak') -or ($path -like '*.trn')) -and (Test-Path -Path $path -PathType Leaf)) {
```

Comment:
Thanks so much for the PR! allcommands.ps1 is compiled at release time and thus this does not need to be edited.